account_due_list/account_move_line.py

```diff
@@ -45,6 +45,66 @@ class AccountMoveLine(models.Model):
                                         compute='_get_invoice',
                                         string='Invoice', store=True)
 
+    maturity_residual = fields.Float(
+        compute='_maturity_residual', string="Residual Amount", store=True,
+        help="The residual amount on a receivable or payable of a journal "
+             "entry expressed in the company currency.")
+
+    @api.multi
+    @api.depends('date_maturity')
```

Comment:
On 15 September 2015 at 09:29, Andrea Cometa <notifications@github.com>
wrote:

> added debit,credit,reconcile_id and reconcile_partial_id deps
>
>
>
Why not the others?